test/integration/drag_and_drop_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'test_helper'
+
+class DragAndDropTest < ActionDispatch::IntegrationTest
+  before do
+    Capybara.current_driver = :poltergeist
+    @user = create :user
+    log_in_as @user
+
+    4.times do |i|
```

Comment:
Anyway, here's a before block that'll work:

```
  before do
    Capybara.current_driver = :poltergeist
    @user = create :user
    log_in_as @user

    4.times do |i|
      pt = create :patient, name: "Patient #{i}",
                       primary_phone: "123-123-123#{i}",
                       created_by: @user
      create :pregnancy, patient: pt
    end

    @user.add_patient Patient.find_by(name: 'Patient 0')
    @user.add_patient Patient.find_by(name: 'Patient 1')
    @user.add_patient Patient.find_by(name: 'Patient 2')
    @user.add_patient Patient.find_by(name: 'Patient 3')
  end
```

Visiting `dashboard_path` then shows you this: 
![image](https://cloud.githubusercontent.com/assets/3866868/18605901/aec93bf8-7c6c-11e6-8f6c-d7dece89913f.png)

gl hf